models/User.js

```diff
@@ -29,6 +29,12 @@ export default model(
         enum: Object.values(genderOptions),
         required: [true, 'Gender is required'],
       },
+      dailyNorma: {
+        type: Number,
+        default: 2000,
+        max: 15000,
```

Comment:
use constant for 15000 here and everywhere else where you use 15000, like validation